src/helpers/OTHelper.js

```diff
@@ -77,8 +79,8 @@ const logRequest = (body) => {
     });
 };
 
-const logOT = (apiKey, sessionId, action, connectionId) => {
-  logRequest(getLog(apiKey, sessionId, action, connectionId));
+const logOT = ({apiKey, sessionId, action, connectionId, proxyUrl}) => {
```

Comment:
Linting: Please add spaces after and before breaks, like this:
{ apiKey, sessionId, action, connectionId, proxyUrl }